docs/source/howto/share_data.rst

```diff
@@ -0,0 +1,248 @@
+
+The script ``many.conf``, instead, defines two different process groups, one for each app.
+So the processes created dynamically by Apache will always be handling one app each.
+The minimal number of Apache daemon processes equals the number of apps, contrarily to the first architecture, where one process is enough to handle two or even a larger number of apps.
+
+Let us call the two apps for this example ``django`` and ``sqlalchemy``.
+In both ``one.conf`` and ``many.conf``, the important directives that should be updated if one changes the paths or names of the apps are:
+
+    - ``WSGIProcessGroup`` to define the process groups for later reference.
+      In ``one.conf`` this directive appears only once to define the generic group ``profiles``, as there is only one kind of process handling both apps.
+      In ``many.conf`` this directive appears once per app and is embedded into a "Location" tag, e.g.::
+
+        <Location /django>
+            WSGIProcessGroup sqlalchemy
+        <Location/>
+
+    - ``WSGIDaemonProcess`` to define the path to the AiiDA virtual environment.
+      This appears once per app in both configurations.
+
+    - ``WSGIScriptAlias`` to define the absolute path of the ``.wsgi`` file of each app.
+
+    - The ``<Directory>`` tag mainly used to grant Apache access to the files used by each app, e.g.::
+
+        <Directory "<aiida.source.code.path>/aiida/restapi/wsgi/app1">
+                Require all granted
+        </Directory>
+
+The latest step is to move either ``one.conf`` or ``many.conf`` into the Apache configuration folder and restart the Apache server.
+In Ubuntu, this is usually done with the commands:
+
+.. code-block:: bash
+
+    cp <conf_file>.conf /etc/apache2/sites-enabled/000-default.conf
+    sudo service apache2 restart
+
+We believe the two basic architectures we have just explained can be successfully applied in many different deployment scenarios.
+Nevertheless, we suggest users who need finer tuning of the deployment setup to look into to the official documentation of `Apache <https://httpd.apache.org/>`_ and, more importantly, `WSGI <wsgi.readthedocs.io/>`__.
+
+The URLs of the requests handled by Apache must start with one of the paths specified in the directives ``WSGIScriptAlias``.
+These paths identify uniquely each app and allow Apache to route the requests to their correct apps.
+Examples of well-formed URLs are:
+
+.. code-block:: bash
+
+    curl http://localhost/django/api/v4/computers -X GET
+    curl http://localhost/sqlalchemy/api/v4/computers -X GET
+
+The first (second) request will be handled by the app ``django`` (``sqlalchemy``), namely will serve results fetched from the profile ``django`` (``sqlalchemy``).
+Notice that we haven't specified any port in the URLs since Apache listens conventionally to port 80, where any request lacking the port is automatically redirected.
```

Comment:
```suggestion
Notice that we have not specified any port in the URLs since Apache listens conventionally to port 80, where any request lacking the port is automatically redirected (port 443 for HTTPS).
```